src/main/java/org/opentestsystem/ap/ims/service/FileUploadService.java

```diff
@@ -86,41 +89,15 @@ public void detachFile(final String itemId, final String fileName) {
         itemRepository.deleteAttachment(user, itemId, fileName);
     }
 
-    public void uploadAslFile(final UploadedFile uploadedFile) {
-        validator.validateAslFile(uploadedFile);
-        uploadFile(uploadedFile, FILE_TYPE_ASL);
-    }
 
-    /**
-     * Validates and processes the braille file upload request.
-     *
-     * @param uploadedFile The upload request holding information like item ID, transaction ID, and the file(s).
-     */
-    public void uploadBrailleFile(final UploadedFile uploadedFile) {
-        validator.validateBrailleFile(uploadedFile);
-        uploadFile(uploadedFile, FILE_TYPE_BRAILLE);
-    }
-
-    public void uploadCcFile(final UploadedFile uploadedFile) {
-        validator.validateCcFile(uploadedFile);
-        uploadFile(uploadedFile, FILE_TYPE_CC);
-    }
-
-    public void uploadFile(final UploadedFile uploadedFile,
-                           final AttachmentFileType attachmentFileType) {
-        final String itemId = uploadedFile.getItemId();
-
-        log.debug("Uploading file for item {}", itemId);
-
-        final MultipartFile file = uploadedFile.getFile();
-
-        Attachment attachment;
-        final AttachmentFile attachmentFile = mapper.newAttachmentFile(file.getOriginalFilename(), attachmentFileType);
-        validator.validateAttachmentFile(itemId, attachmentFile);
-
-        attachment = mapper.newAttachment(attachmentFile);
-
-        final ItemBankUser user = securityUtil.getItemBankUser();
-        itemRepository.saveAttachment(user, itemId, attachment, file);
+    public AttachmentFileType getAttachmentFileTypeForSection(String section) {
+        if(section.equalsIgnoreCase("asl")) {
```

Comment:
instead of using "asl" use SECTION_ASL, these come from ItemConstants.Section is part of the common lilb